src/structures/interfaces/TextBasedChannel.js

```diff
@@ -330,6 +330,44 @@ class TextBasedChannel {
     throw new TypeError('MESSAGE_BULK_DELETE_TYPE');
   }
 
+  /**
+   * Fetches all webhooks for the channel.
+   * @returns {Promise<Collection<Snowflake, Webhook>>}
+   * @example
+   * // Fetch webhooks
+   * channel.fetchWebhooks()
+   *   .then(hooks => console.log(`This channel has ${hooks.size} hooks`))
+   *   .catch(console.error);
+   */
+  fetchWebhooks() {
+    return this.guild.channels.fetchWebhooks(this.id);
+  }
+
+  /**
+   * Options used to create a {@link Webhook} in a {@link TextChannel} or a {@link NewsChannel}.
```

Comment:
We'd have to add voice channels to this, but I suppose we could generalise it instead.

```suggestion
   * Options used to create a {@link Webhook} in a guild text-based channel.
```

Reply:
forum channels can have webhooks; are those "text-based"? maybe just say guild channel

Reply:
That's not in version 13. When it's added, it can be amended.